### PR TITLE
Blog app views

### PR DESCRIPTION
**Pull request by iankiema**
**Changes**

- Implemented initial wireframe designs for the Blog App
- Prioritized functionality over styling in this phase
- Wireframes cover essential pages: All Users, Single User, All Posts by a given User, Single Post
- Leveraged methods in models for effective data processing
- Omitted login logic temporarily; Devise integration planned for upcoming milestones
- Seeded data into the database as per instructions for testing and visualization of posts and comments.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,29 @@
...
+            <div class="post-info">
+              <%= render 'shared/post_summary', post: post %>
+              <p class="card-text"><%= post.text %></p>
+            </div>
+            
+            <div class="com-post">
+                <p class="card-text"><strong>Comments:</strong> <%= post.comments_counter %></p>
+                <p class="card-text"><strong>Likes:</strong> <%= post.likes_counter %></p>
+            </div>
+        </div>
```
Comment (MahmoudBakr23) [inline app/views/posts/index.html.erb]: - [ ] In the post index page, and as per [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md), please make sure that each of the posts in the index page shows the most recent 5 comments. 👍

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,19 @@
...
+            <p class="card-text"><strong>Comments:</strong> <%= @post.comments_counter %></p>
+            <p class="card-text"><strong>Likes:</strong> <%= @post.likes_counter %></p>
+        </div>
+    </div>
+  
+
+    <h2>Comments</h2>
+    <% @post.five_recent_comments.each do | comment |%>
+        <%= render 'shared/comment', comment: comment%>
+    <% end %>
```
Comment (MahmoudBakr23) [inline app/views/posts/show.html.erb]: - [ ] In the post show page, all the comments should be displayed and not only the recent 5 ones, please fix that. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @iankiema 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on working with the controllers and the corresponding views so far
- Nice work with the wireframe
- Descriptive PR and README
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @iankiema 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**